webapp/.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_ENDPOINT=http://localhost:8000
```

Comment:
During deployment, this actually caused a bit of an issue.

Let's keep the ENDPOINT idea but instead apply it in package.json
under the proxy key/value pair

Here's what I had to hack together to get it to work:
```
+  "proxy": "https://localhost:8000/",
   "dependencies": {
     "@material-ui/core": "^4.9.8",
     "@material-ui/icons": "^4.9.1",
diff --git a/webapp/src/actions.js b/webapp/src/actions.js
index f9b89da..ac59b37 100644
--- a/webapp/src/actions.js
+++ b/webapp/src/actions.js
@@ -7,7 +7,7 @@ import {
 } from './action-types';

 export const getCourses = () => createAction({
-  endpoint: `${ENDPOINT}/api/courses/?limit=11000`,
+  endpoint: 'api/courses/?limit=11000',
```

Without this, [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) will fail

Then in the actions file just do endpoint: 'api/courses/'

Reply:
You're only supposed to use that field [during development](https://create-react-app.dev/docs/proxying-api-requests-in-development/), and I don't really want to commit env specific info to Git.

What were the issues during deployment?

Reply:
The issue without this is that nginx ingress even with allowing CORS still fails with this return code:
```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://localhost:8000/api/courses/?limit=11000. (Reason: CORS request did not succeed).
```

I understand it's not exactly ideal for a production environment. But currently, it's the only way I have found that allows all of this to be run in the same docker environment.

Articles/technologies I have tried to get this to work:
https://imti.co/kubernetes-ingress-nginx-cors/
https://pypi.org/project/django-cors-headers/
https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#enable-cors

Reply:
Resolved in DM, if changes need to be made that'll be in another PR.